src/runtime/buffer.h

```diff
@@ -361,6 +361,16 @@ static inline int buffer_strchr(buffer b, int c)
     return -1;
 }
 
+static inline int buffer_strrchr(buffer b, int c)
+{
+    bytes len = buffer_length(b);
+    for (bytes i = 0; i < len; i++) {
```

Comment:
Total nit, but could just loop from len - 1 down to zero

Reply:
I simplified it!